doc/release-notes/PR-10905-OIDC-new-implementation.md

```diff
@@ -0,0 +1,8 @@
+New OpenID Connect implementation including new log in scenarios :ref:`oidc-log-in` for the current JSF frontend, the new Single Page Application (SPA) frontend, and a generic API usage. The API scenario using Bearer Token authorization is illustrated with a Pythons cript that can be found in `bearer-token-example` directory. This Python script prompts you to log in to the Keycloak in a new browser window using selenium. You can run that script with the following commands:
+
+```shell
+    cd bearer-token-example
+    ./run.sh
+```
+
```

Comment:
Somewhere in the release note should we say that the "api-bearer-auth" feature flag has been deprecated and that bearer tokens are always enabled?